siemens/ac_pretty.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# parameters: "ac output directory" "ac total"
+ac_dir=${1}
+ac_total=${2}
+
+echo -----------------------------------------------------------------
+grep "Found" ${ac_dir}'/autocheck_verify.rpt'
+grep "AC Total" ${ac_dir}'/autocheck_verify.rpt'
```

Comment:
This filename is specified many places ([DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)). Please define the report path in a Makefile variable, and then use that everywhere. The parameter to this shell script should be that path instead of `ac_dir`.